deploy/examples/csi/rbd/storageclass.yaml

```diff
@@ -59,8 +59,11 @@ parameters:
   # RBD image format. Defaults to "2".
   imageFormat: "2"
 
-  # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
-  imageFeatures: layering
+  # RBD image features, equivalent to OR'd bitfield value: 63
+  # Available for imageFormat: "2". Older releases of CSI RBD
+  # support only the `layering` feature.  The Linux kernel (KRBD) supports the
+  # full feature complement as of 5.4
+  imageFeatures: layering,fast-diff,object-map,deep-flatten,exclusive-lock
```

Comment:
I'd prefer we have only layering by default and list this alongside in a comment, so user can add it as required.
Most users will still be on lower kernel, they will have to edit this when they get an error after creating a PVC.
This will be very inconvenient according to me.

Reply:
Hrm, my reply seems to have vanished.  I don't think "most" users will be on a pre-5.4 kernel. At some point we have to draw the line, why not now?  It's easy enough for someone to edit, though features can be added per-volume after the fact too via the Ceph CLI.  I'm changing it to as you request; we'll still be improving the outdated info currently in the various files.

Reply:
I've pushed changes, PTAL